src/utils/getDiscourseNodes.ts

```diff
@@ -105,15 +102,15 @@ const getDiscourseNodes = (relations = getDiscourseRelations()) => {
                 : target,
             uid: window.roamAlphaAPI.util.generateUID(),
           })),
-          isRelationBacked: true,
+          source: "relation",
           canvasSettings: {},
         }))
     );
   const configuredNodeTexts = new Set(configuredNodes.map((n) => n.text));
   const defaultNodes = DEFAULT_NODES.filter(
     (n) => !configuredNodeTexts.has(n.text)
   );
-  return defaultNodes.concat(configuredNodes);
+  return configuredNodes.concat(defaultNodes);
```

Comment:
this is the root of the bug.

`findDiscourseNodes` iterates through the nodes to find a matching discourse node. With default ones being defined first, they were matching before configured ones.

We will need a better way to set priorities on nodes that match multiple discourse types. this works for now I think